lib/language_pack/ruby.rb

```diff
@@ -29,7 +29,7 @@ def self.use?
   end
 
   def self.bundler
-    @bundler ||= LanguagePack::Helpers::BundlerWrapper.new.install
+    @@bundler ||= LanguagePack::Helpers::BundlerWrapper.new.install
```

Comment:
We're essentially doing this allready, but without using the extremely confusing class instance variables:

```ruby
  def self.bundler
    @bundler ||= LanguagePack::Helpers::BundlerWrapper.new.install
  end

  def bundler
    self.class.bundler
  end
```

Is there another reason we need this?